src/core_modules/capture-core/components/Pages/MainPage/EventWorkingLists/types/apiTemplate.types.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import { typeof apiAssigneeFilterModes } from '../constants';
+
+export type ApiDataFilterText = {|
+    like: string,
+|};
+
+export type ApiDataFilterNumeric = {|
+    ge?: ?string,
+    le?: ?string,
+|};
+
+export type ApiDataFilterBoolean = {|
+    in: Array<string>,
+|};
+
+export type ApiDataFilterTrueOnly = {|
+    eq: string,
+|};
+
+export type ApiDataFilterDateAbsolute = {|
+    type: 'ABSOLUTE',
+    startDate?: ?string,
+    endDate?: ?string,
+|};
+
+export type ApiDataFilterDateRelative = {|
```

Comment:
i dont think we need export here since this is only used in this file.

Reply:
fixed.